src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -188,7 +188,6 @@ FORBID_C_FUNCTION(void exit(int), "use os::exit");
 FORBID_C_FUNCTION(void _exit(int), "use os::exit");
 FORBID_C_FUNCTION(char* strerror(int), "use os::strerror");
 FORBID_C_FUNCTION(char* strtok(char*, const char*), "use strtok_r");
-FORBID_C_FUNCTION(int sprintf(char*, const char*, ...), "use os::snprintf");
```

Comment:
At some point we would want this and as this should be the last PR to touch hotspot code in this area then this seems like the right PR to have it in. We just need to check that it is actually working as expected.